test/unit_tests/unit_test_helper.cpp

```diff
@@ -328,7 +328,7 @@ namespace Unit_Test
 
     uint32_t runCount()
     {
-        return 1001u; // some magic number for loop. Higher value = higher chance to verify all possible situations
+        return 101u; // some magic number for loop. Higher value = higher chance to verify all possible situations
```

Comment:
Please revert this change and use command line input parameter for the unit test compiled application.